src/Microsoft.DotNet.Build.Tasks.net45/Microsoft.DotNet.Build.Tasks.net45.csproj

```diff
@@ -38,6 +39,7 @@
     <TargetingPackReference Include="System.Threading.Tasks" />
     <TargetingPackReference Include="System.Xml" />
     <TargetingPackReference Include="System.Xml.Linq" />
+    <TargetingPackReference Include="System.Windows.Forms" />
```

Comment:
Ok sounds good I will update to generate the resw file using Xml writer, if he would like to make it full NS, I found that this can be done with MetadataReader rather easier than what we talk the other day, so I could change that as well.